src/components/sections/people/JoiningPane.scss

```diff
@@ -11,4 +11,35 @@
     .ViewSwitch-state-forms:before {
         @include icon($fa-var-cog);
     }
+
+    .RootPaneBase-toolbar {
+        display: flex;
+
+        .viewSwitch {
+            margin-right: 1em;
+            margin-left: 1em;
+            order: 0;
+        }
+        
+        .RootPaneBase-filterButton {
+            margin-left: 0;
+            order: 1;
+        }
+
+        .JoiningPane-addButton {
+            margin-right: 1em;
+            margin-left: auto;
+        }
+    }
+
+    .RootPaneBase-toolbar-column-reverse {
+        display: flex;
+        flex-direction: column-reverse;
+        margin-left: -1em;
+
+        .RootPaneBase-filterButton {
+            margin-top: 0.5em;
+            margin-left: 0.8em;
+        }
+    }
```

Comment:
Is this still needed? Or only part of it?

Reply:
Yeah the toolbar-column-reverse is no longer needed. Fixed.